spec/amp-var-substitutions.md

```diff
@@ -1292,3 +1309,68 @@ Provides the viewport width in pixels available for the page rendering. In contr
   ```
 - **amp-analytics variable**: `${viewportWidth}`
   - Example value: `2560`
+
+#### Analytics Advanced Variable Syntax
+
+The following variables are only supported in `<amp-analytics>`
+
+##### Default
+
+Used to assign fallback value when the variable is resolved to empty string.
+
+- Example: `$DEFAULT(${var}, fallback)`
+
+##### String substr
+
+- Example: `$SUBSTR(${var}, 1, 4)`
+
+##### Trim
+
+- Example: `$TRIM(${var})`
+
+##### ToLowerCase
+
+- Example: `$TOLOWERCASE(${var})`
+
+#### ToUpperCase
+
+- Example: `$TOUPPERCASE(TITLE)`
+
+##### NOT
+
+Used to convert truth-y/false-y value to `'false'`/`'true'` string.
+
+- Example `$NOT(hello)` = `'false'` and `$NOT()` = `'true'`
+
+##### Base64
+
+- Example `$BASE64(PAGE_VIEW_ID)`
+
+##### Hash
+
+SHA-384 hash
+
+- Example: `$HASH(PAGE_VIEW_ID)`
+
+##### If
+
+An if statement to choose between values
+
+- Example: `$IF(${var}, value1, value2)`
+- Example result: Resolve to `value2` if `${var}` resolves to falsey value including `'false'`, `''`, `'0'`, `'null'`, `'Nan'` or `'undefined'`.
```

Comment:
nit: Remove `Example result` as we don't use it anywhere else:

```
Example: `$IF(${var}, value1, value2)` resolves to `value2` if `${var}` resolves to falsey value (`'false'`, `''`, `'0'`, `'null'`, `'Nan'` or `'undefined'`), otherwise will resolve to `value1`.
```